egs/tedlium/s5/local/confidence_calibration.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+. cmd.sh
+. path.sh
+
+# Global options,
+graph=exp/tri3/graph
+arpa_gz=db/cantab-TEDLIUM/cantab-TEDLIUM-pruned.lm3.gz
+lmwt=13
+
+# Dev-set options,
+dev_data=data/dev
+dev_latdir=exp/tri3_mmi_b0.1/decode_dev_it4
+
+# Eval-set options,
+eval_data=data/test
+eval_latdir=exp/tri3_mmi_b0.1/decode_test_it4
+
+. utils/parse_options.sh
+set -euxo pipefail
+
+# Derived options,
+dev_caldir=$dev_latdir/confidence_$lmwt
+eval_caldir=$eval_latdir/confidence_$lmwt
+
+###### Train the calibration,
+steps/conf_train_calibration.sh --cmd "$decode_cmd" --lmwt $lmwt \
+  $dev_data $graph $arpa_gz $dev_latdir $dev_caldir
+
+###### Apply the calibration to eval set,
+steps/conf_apply_calibration.sh --cmd "$decode_cmd" \
+  $eval_data $graph $arpa_gz $eval_latdir $dev_caldir $eval_caldir
+# The final confidences are here '$eval_caldir/ctm_calibrated',
+
+###### Sclite scoring,
+# We will produce NCE which shows the ``quality'' of the confidences.
+# Please compare with the default scoring script for your database.
+
+# Scoring tools,
+hubscr=$KALDI_ROOT/tools/sctk/bin/hubscr.pl 
+hubdir=`dirname $hubscr`
+
+# Inputs,
+ctm=$eval_caldir/ctm_calibrated
+stm=$eval_data/stm
+glm=$eval_data/glm
+
+# Normalizng CTM, just like in 'local/score_sclite.sh',
```

Comment:
Hi, in tedlium the 'glm' is a dummy file, but I believe the scoring should be the same as in local/score.sh, so we can compare the scores with normal setup. The 'grep-pattern' is now removed, but I keep filtering words like '[...],<...>,%...,...-' because these are never used for normal words and it was improving the NCE on swbd by 0.004.